src/authorization-request/AuthorizationRequest.ts

```diff
@@ -1,4 +1,7 @@
 import { JWTVerifyOptions } from 'did-jwt';
+import { decodeJWT } from 'did-jwt';
+import { JWTDecoded } from 'did-jwt/lib/JWT';
+import forge from 'node-forge';
```

Comment:
This cannot work. This library is used in browser, RN and Node. The lib is node only